cmd/event-received/sirius_event_handler.go

```diff
@@ -432,3 +445,27 @@ func handleChangeConfirmed(ctx context.Context, client dynamodbClient, certifica
 
 	return nil
 }
+
+func handleCertificateProviderIdentityCheckedFailed(ctx context.Context, dynamoClient dynamodbClient, notifyClient NotifyClient, bundle Bundle, appPublicURL string, event *events.CloudWatchEvent) error {
+	var v uidEvent
+	if err := json.Unmarshal(event.Detail, &v); err != nil {
+		return fmt.Errorf("failed to unmarshal detail: %w", err)
+	}
+
+	provided, err := getDonorByLpaUID(ctx, dynamoClient, v.UID)
+	if err != nil {
+		return fmt.Errorf("failed to get donor: %w", err)
+	}
+
+	if provided.Tasks.SignTheLpa.IsCompleted() {
```

Comment:
I don't think we need this check, as we only send paper forms once the donor signs

Reply:
Good point, my head is always in online world.